src/routes/retro-certs.js

```diff
@@ -9,6 +9,39 @@ const weeksCompleted = require("../utils/checkFormData");
 function createRouter() {
   const router = Router();
 
+  async function staffViewAuthStatus(postJson, responseJson) {
+    responseJson.status = AUTH_STRINGS.statusCode.userNotFound;
+    const [userRecord] = await Promise.all([
+      cosmos.getUserByNameDobSsn(
+        postJson.lastName || "",
+        postJson.dob,
+        postJson.ssn
+      ),
+    ]);
+    if (!userRecord) {
+      console.log("failed staff view login");
+      return responseJson;
+    }
+
+    console.log("staff view login", userRecord.id);
+    userRecord.staffLastViewedAt = Date.now();
```

Comment:
I wonder if this should be an array. You could maybe limit it to a max of 10 entries or something so it doesn't grow too big.

Reply:
Yeah... I think it's fine to just record the last login. EDD has approved that as well.